src/components/NavBar/NavBar.js

```diff
@@ -12,16 +12,19 @@ const NavBar = () => {
 
   return (
     <nav className="nav-bar">
-      <div class="container" id="menu" onClick={toggleMenu}>
-        <div class="bar1"></div>
-        <div class="bar2"></div>
-        <div class="bar3"></div>
+      <div className="container" id="menu" onClick={toggleMenu}>
```

Comment:
I think I need an aria label here for the role of the hamburger menu.

Reply:
I haven't tried a hamburger menu yet ( I might for this project).  You could always run a screenreader through the app and see where it's not describing as you want.

Reply:
Great idea, will do, thank you!